.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,67 @@
+name: Build and Deploy Docs
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build-and-deploy:
+    name: Build and Deploy
+    runs-on: ubuntu-20.04
+    env:
+      MIKE_VERSION: v3
+      MIKE_ALIAS: latest
+      DOCKER_IMAGE: squidfunk/mkdocs-material:8.5.9
+      GH_NAME: "OpenUI5 Bot"
+      GH_EMAIL: "openui5@sap.com"
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install npm dependencies
+      run: npm ci
+
+    - name: Update CLI Doc
+      run: npm run generate-cli-doc
+
+    - name: GitHub Configuration
+      run: |
+        git fetch origin gh-pages --depth=1
+        git config --local user.email $GH_EMAIL
+        git config --local user.name $GH_NAME
+
+    - name: Build Mike Image
+      run: |
+        chmod +x ./scripts/buildImage.sh
```

Comment:
Please do that locally, so that it can also be executed locally. A file in git also contains the execution flag.

Reply:
Thanks a lot! Didn't know that :)

Reply:
Please also remove this line then.